### PR TITLE
Add AdminCommissioningCluster

### PR DESCRIPTION
**Pull request by mfucci**
Required for iOS / Android support

### CODE REVIEW

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TlvByteString, TlvField, TlvObject, TlvUInt16, TlvUInt32 } from "@project-chip/matter.js";
+import { Cluster, Command, TlvNoArguments, TlvNoResponse, OptionalCommand } from "./Cluster";
+
+export const AdminCommissioningCluster = Cluster({
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:10]: Add splec ref Core specs 11.18

Comment (Apollon77) [reply src/matter/cluster/AdminCommissioningCluster.ts:10]: This cluster is used to trigger a Node to allow a new Administrator to commission it. It defines Attributes, Commands and Responses needed for this purpose.

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    id: 0x3c,
+    name: "AdministratorCommissioning",
+    revision: 1,
+
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:14]: The cluster attributes are missing, maybe leave a "TODO" to be added later? maybe reference to 11.18.7

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    commands: {
+        openCommissioningWindow: Command(0, TlvObject({
+            commissioningTimeout: TlvField(0, TlvUInt16),
+            pakeVerifier: TlvField(1, TlvByteString),
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:18]: pakePasscodeVerifier ?

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            discriminator: TlvField(2, TlvUInt16),
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:19]: Limited 0 to 2047

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+            iterations: TlvField(3, TlvUInt32),
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:20]: Limited 1000 to 100000

File: src/matter/cluster/AdminCommissioningCluster.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+            salt: TlvField(4, TlvByteString),
```
Comment (Apollon77) [inline src/matter/cluster/AdminCommissioningCluster.ts:21]: Length 16 to 32

**Review by Apollon77 — APPROVED** [review]
Basically minimalstic implementation ... but I would add some more todos (if you like to iterate fast and do it perfect later)

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 1 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:10]: Added

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 3 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:18]: Done

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 4 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:19]: Done

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 5 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:20]: Done

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 6 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:21]: Done

File: src/matter/cluster/AdminCommissioningCluster.ts (hunk 2 above)
Comment (mfucci) [reply src/matter/cluster/AdminCommissioningCluster.ts:14]: Done

**Comment by mfucci** [comment]
Comments addressed

**Review by Apollon77 — APPROVED** [review]

**Pull request merged**